addons/escoria-core/game/core-scripts/esc/commands/transition.gd

```diff
@@ -0,0 +1,37 @@
+# `transition transition_name in|out`
+#
+# Performs a fade in or fade out transition manually.
+# 
+# @ESC
+extends ESCBaseCommand
+class_name FadeCommand
+
+
+# Return the descriptor of the arguments of this command
+func configure() -> ESCCommandArgumentDescriptor:
+	return ESCCommandArgumentDescriptor.new(
+		2, 
+		[TYPE_STRING, TYPE_STRING],
+		[null, null]
+	)
+	
+	
+# Validate wether the given arguments match the command descriptor
+func validate(arguments: Array):
+	if not arguments[0] in ["in", "out"]:
```

Comment:
This should be arguments[1]. arguments[0] is the transition name.

Additionally, you could check if the transition name is valid.

Reply:
Done